client/src/plus/app.tsx

```diff
@@ -1,56 +1,65 @@
-import React, { useEffect } from "react";
-import { useLocale } from "./../hooks";
+import React from "react";
+import Modal from "react-modal";
+import { useLocale } from "../hooks";
 
 import "./fonts/metropolis.css";
 import "./fonts/inter.css";
 import "./index.scss";
 
+Modal.setAppElement("main");
+
+function BackToMDNModal({ onRequestClose }) {
+  const locale = useLocale();
+
+  return (
+    <Modal
+      overlayClassName="modal"
+      className="modal-inner"
+      aria-expanded
+      onRequestClose={onRequestClose}
+      isOpen
+      parentSelector={() => document.querySelector(".plus main")!}
+    >
+      <div className="modal-inner">
+        <div className="modal-content">
+          <figure className="dino" />
```

Comment:
In there a reason we are using a `figure` element here as opposed to an `img` element. Generally, I would just go with an `img` if you are using the `figure` to wrap some graphical image that will also benefit from a `figcaption`. I looks like this is just presentational so:

1. Could we just set a background image on the `model-content` element?
2. Use a `img` element with an empty `alt`?